
This fixes the nice library function for architectures without nice syscal.
AFAICS x86-64 is one.

  - Rene Rebe <reneÂ@exactcode.de>

--- dietlibc/lib/__nice.c.vanilla	2000-01-01 00:00:00.000000000 +0100
+++ dietlibc/lib/__nice.c	2005-01-14 03:45:41.386231296 +0100
@@ -4,6 +4,6 @@
 
 #ifndef __NR_nice
 int nice(int i) {
-  return setpriority(getpriority(PRIO_PROCESS,0)+i);
+  return setpriority(PRIO_PROCESS,0,getpriority(PRIO_PROCESS,0)+i);
 }
 #endif
